test/plugin/scenarios/shardingsphere-5.0.0-scenario/support-version.list

```diff
@@ -14,4 +14,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-5.0.0-beta
+5.0.0
```

Comment:
Is `beta` still supported? Or it is out of the supported list. Either way, please make it clear in the change log.

Reply:
Beta can be supported too.

Reply:
However, we don't recommend the beta version.

Reply:
OK, just make sure it is clear in the change log. The version recommendation is your community's call for sure.